### DISCUSSION TITLE
How does the owner() call send the ETH into a treasury instead of the contract owner’s address?

### DISCUSSION THREAD

**Discussion by cdt-eth** [nounsDAO/nouns-monorepo#405] [Solidity / smart contracts] [answered]
How does the owner() call send the ETH into a treasury instead of the contract owner’s address?

**Comment by solimander** [comment, marked as answer]
The Nouns DAO is the contract owner. In order to give the DAO full governance over the ecosystem, it was made the owner of all the protocol contracts immediately after launch.
